### DISCUSSION TITLE
I have an online store, how to quick install scandipwa?

### DISCUSSION THREAD

**Discussion by AngeloZou** [scandipwa/scandipwa#2509] [Install & Upgrade] [answered]
Hi,
I have an online store, how to quick install scandipwa.

My system is as follows:
OS: CentOS Linux 7 (Core)
Kernel: 3.10.0-1160.21.1.el7 
Panel: cPanel 11.94.0.4
Web Server: Apache 2.4.46
PHP Version: 7.3.27
Database: MySQL 10.3.28-Mari 
Software: elasticsearch

Kind regards,
Angelo

**Comment by lianastaskevica** [comment, marked as answer]
Hi @AngeloZou! Generally, you can find all our setup instructions here - https://docs.scandipwa.com/getting-started/getting-started.

If you have an existing Magento 2 store, this is the guide you will need - https://docs.scandipwa.com/getting-started/getting-started/magento-integration.
